ServerCore/PuzzleStateHelper.cs

```diff
@@ -569,6 +569,14 @@ public static async Task UpdateTeamsWhoSentResponse(PuzzleServerContext context,
 
                     await context.SaveChangesAsync();
                     transaction.Commit();
+
+                    var teamMembers = await (from TeamMembers tm in context.TeamMembers
+                                             join Submission sub in context.Submissions on tm.Team equals sub.Team
+                                             where sub.Puzzle == response.Puzzle && sub.SubmissionText == response.SubmittedText
+                                             select tm.Member.Email).ToListAsync();
+                    MailHelper.Singleton.SendPlaintextBcc(teamMembers,
+                        $"Response updated for '{response.SubmittedText}' on {response.Puzzle.Name}",
+                        $"The new response for this submission is: '{response.ResponseText}'.");
```

Comment:
Should we include a bonus comment if the submissions was changed to an answer?

Reply:
Nah, the response will make that clear. "Correct!"